src/Module.Server/ds_config_crpg_dtv.txt

```diff
@@ -9,7 +9,7 @@ disable_culture_voting
 CultureTeam1 vlandia
 CultureTeam2 battania
 RoundPreparationTimeLimit 2
-MapTimeLimit 60
+MapTimeLimit 150
```

Comment:
The limit is hardcoded to 60

Reply:
Is there another way to extend it?

Reply:
I'll look into using ```TimerComponent``` to increase the remaining time after each round is cleared. Similar to how Conquest handles new phases.

Reply:
Yes that would work.